third-party/realdds/src/dds-stream.cpp

```diff
@@ -4,10 +4,16 @@
 #include <realdds/dds-stream.h>
 #include "dds-stream-impl.h"
 
+#include <realdds/dds-topic.h>
+#include <realdds/dds-topic-reader.h>
+#include <realdds/dds-subscriber.h>
+#include <realdds/topics/image/image-msg.h>
 #include <realdds/dds-exceptions.h>
 
+#include <fastdds/dds/subscriber/SampleInfo.hpp>
 
-namespace realdds {
+using namespace eprosima::fastdds::dds;
```

Comment:
Any particular reason? You don't seem to be making use of it, either...

Reply:
Removed